lib/shelley/test/data/cardano-node-shelley/genesis.yaml

```diff
@@ -32,7 +31,8 @@ updateQuorum: 5
 maxMajorPV: 25446
 initialFunds: {}
 maxLovelaceSupply: 45000000000000000
-networkMagic: 1
+protocolMagicId: 1
+networkMagic: 764824073
```

Comment:
why we need this?

Reply:
While new Shelley addresses uses the new protocolMagic (`1`), the **network** still uses the old magic (`764824073`). This is therefore needed to match the right network configuration of the cluster.